### PR TITLE
Testsetup

### PR DESCRIPTION
**Pull request by FritzHoing**
Fixes the playwright config and adds aria-label as selectors.

### CODE REVIEW

File: e2e-tests/auth.setup.ts [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-  + 'code&scope=openid&nonce=6cd0da9a-b45c-4a9e-931d-99b6725a148e');
+  await page.goto(`https://${process.env.HOST}/auth/realms/SHOGun`
+    + '/protocol/openid-connect/auth?client_id=shogun-client'
+    + `&redirect_uri=https%3A%2F%2F${process.env.HOST}%2Fclient%2F%3FapplicationId%3D21&state=9a983abe-3b0c-`
+    + '41cb-9b7e-1d9120956959&response_mode=fragment&response_type'
+    + '=code&scope=openid&nonce=72884466-0535-4a24-8c15-9e7f14d88a65');
 
+  // @ts-ignore
   await page.getByLabel('Username or email').fill(`${process.env.ADMIN_LOGIN}`);
+  // @ts-ignore
```
Comment (LukasLohoff) [inline e2e-tests/auth.setup.ts]: I think these might not be necessary, can you check if you still need the `@ts-ignore`?

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -70,7 +70,9 @@ export const Header: React.FC<HeaderProps> = ({
     const items = [
       <SearchField
         key="search"
-        aria-label="search-bar"
+        aria-label='fritz'
```
Comment (LukasLohoff) [inline src/components/Header/index.tsx:73]: was this intended?

File: src/e2e-tests/test-results/auth.setup.ts-authenticate-as-admin-setup-retry1/trace.zip
Comment (LukasLohoff) [inline src/e2e-tests/test-results/auth.setup.ts-authenticate-as-admin-setup-retry1/trace.zip:1]: do we need the /test-result files or can we add them to `.gitignore`?

File: src/components/Header/index.tsx (hunk 2 above)
Comment (FritzHoing) [reply src/components/Header/index.tsx]: Good catch! It is adjusted now.

File: e2e-tests/auth.setup.ts (hunk 1 above)
Comment (FritzHoing) [reply e2e-tests/auth.setup.ts]: You are right, i have removed the ts-ignore!

**Comment by FritzHoing** [comment]
Thank you very much for the review @LukasLohoff !

File: playwright.config.js [hunk 3]
```diff
@@ -17,29 +18,30 @@ export default defineConfig({
     open: 'never'
   }]],
   use: {
-    baseURL: process.env.HOST,
+    // das mal ausprobieren:
```
Comment (dnlkoch) [inline playwright.config.js:21]: This comment can be removed I assume?

File: src/e2e-tests/userMenu.spec.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { test } from '@playwright/test';
+
+import { userMenu } from '@terrestris/shogun-e2e-tests/dist/shogun-gis-client/header/userMenu';
+
+test.use({
+  storageState: 'playwright/.auth/admin.json'
+});
+
+test('test', async ({
```
Comment (dnlkoch) [inline src/e2e-tests/userMenu.spec.ts:9]: We should apply some useful names to the test (here and in all others).

File: tsconfig.json [hunk 5]
```diff
@@ -104,6 +102,9 @@
     "**.config.js"
   ],
   "include": [
-    "./src"
-  ]
+    "./src",
+    "e2e-tests/annotation.spec.ts",
+    "node_modules/@terrestris/shogun-e2e-tests/e2e-tests/shogun-gis-client/header/header",
```
Comment (dnlkoch) [inline tsconfig.json:107]: Is this needed?

File: tsconfig.json [hunk 6]
```diff
@@ -104,6 +102,9 @@
...
+    "e2e-tests/annotation.spec.ts"
```
Comment (dnlkoch) [inline tsconfig.json:108]: Duplicate entry. (Is this needed anyway?)

**Pull request merged**